pom.xml

```diff
@@ -28,10 +28,10 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-data-jpa</artifactId>
         </dependency>
-        <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-oauth2-client</artifactId>
-        </dependency>
+<!--        <dependency>-->
+<!--            <groupId>org.springframework.boot</groupId>-->
+<!--            <artifactId>spring-boot-starter-oauth2-client</artifactId>-->
+<!--        </dependency>-->
```

Comment:
why this comment out ?

Reply:
It was commented out in push #36-html-home that I prepared for Dennis on Friday(?). With those dependencies "on" we were redirected to login straight away. Instead of pulling that request for use with oautl2, it was merged to main. So it ended up in master. Shell I comment it out now, can do it probably directly on github?